aptos-move/block-executor/src/scheduler.rs

```diff
@@ -570,6 +571,52 @@ impl Scheduler {
         SchedulerTask::NoTask
     }
 
+    pub fn finish_execution_during_commit(&self, txn_idx: TxnIndex, revalidate_suffix: bool) {
```

Comment:
we don't need to take revalidate_suffix here, let's decrease validation index every time on commit reexecution (and no need to decrease after failed validation in commit)